### ISSUE TITLE
Airdrop-ng /usr/local/lib/python2.7/dist-packages/airdrop/libDumpParse.py Error in line 23;

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1593]
**Reported by JimvanDam on 29 Jun 2015 10:54 UTC**

I installed airdrop-ng in Kali Linux 1.1.0a and I got this error:

```
airdrop-ng -i mon0

#################################################
#             Welcome to AirDrop-ng             #
#################################################

Traceback (most recent call last):
  File "/usr/local/bin/airdrop-ng", line 1038, in <module>
    Targeting.run() 
  File "/usr/local/bin/airdrop-ng", line 815, in run
    self.targets = self.dataParse()
  File "/usr/local/bin/airdrop-ng", line 806, in dataParse
    parsedAiro = self.AirParser.parser(self.Airo)
  File "/usr/local/lib/python2.7/dist-packages/airdrop/libDumpParse.py", line 11, in parser
    fileOpenResults = self.airDumpOpen(file)
  File "/usr/local/lib/python2.7/dist-packages/airdrop/libDumpParse.py", line 20, in airDumpOpen
    openedFile = open(file, "r")
TypeError: coercing to Unicode: need string or buffer, NoneType found
```

I fixed it by adding a line before line 23 defining "file", like so:

```
                try:
                        file = "/root/rules.txt"
                        openedFile = open(file, "r")
                except IOError:
                        print "Error Airodump File",file,"does not exist"
                        Exit(1)
```

This works. It outputs "Error Airodump File "/root/rules" does not exist" now, and not an error (This is an error too of course, but hey, it works). This is not the best fix there is, but this solves my problem.

I hope this will be fixed soon.

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 29 Jun 2015 10:54 UTC**

You did not give a `<pcap file>` to write to. See docs: http://www.aircrack-ng.org/doku.php?id=airdrop-ng

I.E `airdrop-ng [options] <pcap file>`

**Comment by aircrack-ng** [comment]
**Comment by misterx on 29 Jun 2015 10:54 UTC**

Should be fixed in aircrack-ng/aircrack-ng@a4249c49d75ba0a6cebf0857eb3a8f2944ebad1d and aircrack-ng/aircrack-ng@d396914f6b056bab49e8a67f18ca5cb2e0f42beb.
